packages/composites/por-indexer/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# @chainlink/por-indexer-adapter
+
+## 1.0.0
+
+### Major Changes
+
+- EAv2 Release. Start of individual EA versioning.
```

Comment:
Maybe try checking if you should be using a changeset here instead of manually starting the changelog

Reply:
Gonna do this after the PoR rearch PR is merged.

Reply:
Retracting my comment here. Doing it this way is completely fine, resolving this.